lib/swig/structs.i

```diff
@@ -0,0 +1,252 @@
+typedef struct{
+} encrypt__Sha256Xor;
+
+typedef struct{
+    GoInt_ N;
+    GoInt_ R;
+    GoInt_ P;
+    GoInt_ KeyLen;
+} encrypt__ScryptChacha20poly1305;
+
+typedef struct{
+    GoUint32_  n[10];
+} secp256k1go__Field;
```

Comment:
Do not duplicate these . Use [SWIG `%include` directive](http://www.swig.org/Doc1.3/Preprocessor.html#Preprocessor_nn2) .

Put this in `skycoin.cipher.secp256k1go.i`